mantidproject/mantid
# Run Workbench Tests with Qt5

**Description of work.**

Run the workbench unit tests against Qt5 and mantidqt tests against Qt4 and Qt5.

**To test:**

Code review and check there are now `mantidqt_qt4` and `mantidqt_qt5` tests available.

*No issue associated*

Does this update require release notes?

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
